core/src/main/java/org/eclipse/hono/util/CredentialsConstants.java

```diff
@@ -116,6 +116,14 @@
      * The vert.x event bus address to which inbound credentials messages are published.
      */
     public static final String EVENT_BUS_ADDRESS_CREDENTIALS_IN = "credentials.in";
+    /**
+     * The regular expression to validate that the auth-id field supplied in credentials is legal.
+     */
+    public static final String AUTH_ID_VALUE_REGEX = "^[a-z0-9-=]+$";
```

Comment:
Maybe you can define default reges at one place, like `DEFAULT_ID_REGEX` and reuse that value.

Reply:
I moved the regexp for tenant id and device id to `RegistryManagementConstants.DEFAULT_ID_REGEX`